### ISSUE TITLE
``require('lspconfig').bashls.setup {}`` runs commands in PATH on autocomplete

### ISSUE THREAD

**Issue by ChillerDragon** [nvim-lua/kickstart.nvim#727]
# Probably out of scope

Sorry I did not fully investigate yet who's responsibility it is to fix this bug. There are so many projects working together to cause this bug. It seems like it is not kickstart feel free to close the issue if it is out of scope.

It is also not the default kickstart config but it is a +1 line change.

Maybe this should be a bug report in https://github.com/bash-lsp/bash-language-server instead. Or in https://github.com/neovim/nvim-lspconfig

# Whats the issue

Adding ``require('lspconfig').bashls.setup {}`` to init.lua runs programs on auto complete.

Here is an example of me typing teew in a shell file. And it runs ``teeworlds`` which is in my PATH. So it auto executes the SDL game [teeworlds](https://teeworlds.com/) which is quite distracting because it is a GUI app. I assume it also run the ``tee`` command at some point but it did not give visual feedback. That seems highly dangerous in general. And in case of GUI apps its mostly annoying. It runs the suggested completion as a command without even the user accepting the completion. I did not test but I assume typing ``rebo`` and it automatically running ``reboot`` could be quite annoying (**EDIT:** I tested it. Seems like reboot is not being triggered).

https://github.com/nvim-lua/kickstart.nvim/assets/20344300/03333a4b-e18f-4659-b1e1-7010f24366be

Maybe someone ([bash-language-server](https://github.com/bash-lsp/bash-language-server) ??) runs completion from PATH plus --help for the documentation note?

**Comment by dam9000** [comment]
wow this looks dangerous indeed, I would suggest opening an issue with bash-language-server, I think they would be more likely able to help than here.

**Comment by ChillerDragon** [comment]
> wow this looks dangerous indeed, I would suggest opening an issue with bash-language-server, I think they would be more likely able to help than here.

Thanks for pointing me to the right place.


I just collected more evidence for my --help theory. Typing ``wireshark`` launches nothing. But also ``wireshark --help`` launches nothing. So I created a file in /usr/local/bin with the name ``yireshark`` and the content is just ``wireshark`` so thats basically a shell script that launches wireshark and ignores the --help flag. Then typing yire in neovim launches wireshark indeed.

![image](https://github.com/nvim-lua/kickstart.nvim/assets/20344300/5e3e5484-d28f-464c-ba65-7c3af92a82e0)


If my theory is correct every tool that ignores --help is just launching on key press.
